src/modules/emote_autocomplete/components/AutocompletePopover.jsx

```diff
@@ -0,0 +1,58 @@
+import React, {useEffect, useRef, useState} from 'react';
+import classNames from 'classnames';
+import Popover from 'rsuite/Popover';
+import mergeRefs from 'react-merge-refs';
+import styles from './AutocompletePopover.module.css';
+import Emotes from './Emotes.jsx';
+import repositionPopover from '../../../utils/popover.js';
+import useResize from '../../../common/hooks/Resize.jsx';
+
+const TOP_PADDING = 2;
+const DEFAULT_POPOVER_WIDTH = 300;
```

Comment:
why 300? should the popover be more dynamically in width? have we considered different sized embeds?